exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -223,19 +223,65 @@ public void connect(Properties props) throws RpcException {
     connect(null, props);
   }
 
+  /**
+   * Function to populate the endpointList with information of all the drillbits
+   * provided in the connection string by client
+   * @param endpointList - ArrayList of DrillbitEndpoints
+   * @param drillbits - One or more drillbit ip[:port] provided in connection string
```

Comment:
Changing the param name from `drillbit` to `drillbits` will have a doc impact. The current documentation specifies the usage of `drillbit` but allows for a CSV list 
The convention for zookeepers also is to just specify 
`zk=<node1[:port]>[,node2[:port]]`